### ISSUE TITLE
NC 14.0.2 RC2 - still wrong checks ...

### ISSUE THREAD

**Issue by abysso2** [nextcloud/server#11748]
### Steps to reproduce
1. Upgrade to NC 14.0.2 RC2
2. Go to settings - Übersicht

### Expected behaviour
Tell us what should happen

No check should alert

### Actual behaviour
Tell us what happens instead

Es gibt einige Warnungen bei Deiner Systemkonfiguration.

    Dein Web-Server ist nicht richtig eingerichtet um "/.well-known/caldav/" aufzulösen. Weitere Informationen findest Du in der Dokumentation.
    Dein Web-Server ist nicht richtig eingerichtet um "/.well-known/carddav/" aufzulösen. Weitere Informationen findest Du in der Dokumentation.

### Server configuration

**Operating system**:
Centos 7.5

**Web server:**
ngninx 1.12.2-2

    location = /.well-known/carddav {
      return 301 $scheme://$host/remote.php/dav;
    }

    location = /.well-known/caldav {
      return 301 $scheme://$host/remote.php/dav;
    }

**Database:**

mariadb 10.3.10
**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)
14.0.2 RC2

**Updated from an older Nextcloud/ownCloud or fresh install:**
Update from 14.0.2 RC1 (before 13.0.6, 14.0.1)

```

**Comment by timm2k** [comment]
Is there may any rule that rejects access to .files oder .directories in your config?

Something like this would break support to .well-known and chunked uploads:
```
# Disallow access to hidden files and directories
location ~ /\. {
     return 404;
}
```

**Comment by BernieO** [comment]
Could you post the output of those two commands:
```
curl -iLu USERNAME:PASSWORD https://DOMAIN.COM/.well-known/caldav
curl -iLu USERNAME:PASSWORD https://DOMAIN.COM/.well-known/carddav
```
If you use a selfsigned certificate you might have to use `-kiLu` instead of `-iLu`

**Comment by MorrisJobke** [comment]
Fixed in #11738 - will be shipped in 14.0.2
